src/commons/BrowserUtils.ts

```diff
@@ -32,6 +32,34 @@ export interface IComparisonPath {
   diffPath: string;
 }
 
+const sanitizeString = (str = '') =>
+  str
+    .trim()
+    .replace(/[^a-z0-9]/gi, '_')
+    .toLowerCase();
+
+//this is working because we are setting those values in a beforeTest hook in wdio config
+const getFilename = (filenameSuffix) => {
```

Comment:
1. isn't it better to move it to TestUtils.ts?
2. it's not typescript

Reply:
moved and added types